libs/ballot-interpreter/src/interpret.ts

```diff
@@ -503,17 +492,18 @@ export async function convertRustInterpretResult(
  */
 function validateInterpretResults(
   results: SheetOf<InterpretFileResult>,
-  {
-    electionHash,
+  options: InterpreterOptions
+): SheetOf<InterpretFileResult> {
+  const {
+    electionDefinition: { electionHash },
     precinctSelection,
     testMode,
-  }: {
-    electionHash: string;
-    precinctSelection: PrecinctSelection;
-    testMode: boolean;
-  }
-): SheetOf<InterpretFileResult> {
-  return mapSheet(results, ({ interpretation, normalizedImage }) => {
+  } = options;
+
+  return mapSheet(results, (result) => {
+    const { normalizedImage } = result;
+    const interpretation = normalizeBallotMode(result.interpretation, options);
```

Comment:
Why normalize the test mode rather than just allowing both when the flag is set?

Reply:
Yeah, good question - started out with just skipping the test mode validation, but wasn't sure if the `isTestMode` metadata on the eventual CVR would be used anywhere else downstream. Noticed the NH HMPB code path sets the ballot's `isTestMode` based on the interpreter's test mode and went that route to be safe, but maybe it's not an issue and we can use the simpler approach here. Do you have any context here?

cc @arsalansufi - wondering if we're there are any downstream side-effects to having non-test-mode CVRs in test-mode CVR exports (guessing we only ever reference the test mode on the batch export metadata in the new continuous export world?)

Reply:
+1 -- also leaning this way after thinking about it some more. Don't foresee a need to know the original physical ballot mode later down the line and if we do end up wanting that, we could add more specific metadata